src/CardClass.js

```diff
@@ -0,0 +1,49 @@
+export default class Card {
```

Comment:
I guess it's better to name it just `Card.js` no need for word `Class`.
There is a convention that upper cased files are always classes